foreman/data_refinery_foreman/gatherer/agents/microarray_ae.py

```diff
@@ -0,0 +1,137 @@
+"""
```

Comment:
can you change the names of these files to match the class name? ae_agent.py etc?